src/bot-events.ts

```diff
@@ -2,9 +2,22 @@ import { LogNewAnswer, ProposalQuestionCreated } from "./services/reality";
 import { Space } from "./types";
 
 export const enum BotEventNames {
-  START = "start",
+  // General
+  STARTED = "started",
+  PROCESSING_ENDED = "processing_ended",
```

Comment:
This is the new event. It signals when the main `while` loop, which keep the application fetching new blocks forever, exits. After it, we can proceed to close all the dependencies (like the db connection)